src/main/java/com/sparta/popupstore/domain/review/entity/Review.java

```diff
@@ -37,4 +38,9 @@ public Review(Long id, PopupStore popupStore, User user, String contents, int st
         this.contents = contents;
         this.star = star;
     }
+
+    public void update(ReviewCreateRequestDto requestDto) {
+        star = requestDto.getStar();
+        contents = requestDto.getContents();
+    }
```

Comment:
```suggestion
    public void update(String contents, int star) {
        this.contents = contents();
        this.star = star;
    }
```
entity에서 dto를 파라미터로 받으면 좋지 않습니다.